### PR TITLE
Add effectiveness checker chinese translation

### PR DESCRIPTION
**Pull request by zx900930**

### CODE REVIEW

File: e7-damage-calc/zhTW/effectiveness.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="zhTW">
+    <head>
+        <script defer src="https://www.googletagmanager.com/gtag/js?id=UA-147508975-1"></script>
+        <script>
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+
+          gtag('config', 'UA-147508975-1', { 'page_title' : 'effectiveness', 'page_path': '/e7-damage-calc/effectiveness' });
```
Comment (maphe) [inline e7-damage-calc/zhTW/effectiveness.html:10]:
```suggestion
          gtag('config', 'UA-147508975-1', { 'page_title' : 'effectiveness', 'page_path': '/e7-damage-calc/zhTW/effectiveness' });
```

File: e7-damage-calc/zh/effectiveness.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="zhCN">
+    <head>
+        <script defer src="https://www.googletagmanager.com/gtag/js?id=UA-147508975-1"></script>
+        <script>
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+
+          gtag('config', 'UA-147508975-1', { 'page_title' : 'effectiveness', 'page_path': '/e7-damage-calc/effectiveness' });
```
Comment (maphe) [inline e7-damage-calc/zh/effectiveness.html:10]:
```suggestion
          gtag('config', 'UA-147508975-1', { 'page_title' : 'effectiveness', 'page_path': '/e7-damage-calc/zh/effectiveness' });
```

Comment (zx900930) [reply e7-damage-calc/zh/effectiveness.html]: Fixed https://github.com/maphe/maphe.github.io/pull/39/commits/4f92bb0f86b6a0286ceb9256273be8531e1b4cea

File: e7-damage-calc/zhTW/effectiveness.html (hunk 1 above)
Comment (zx900930) [reply e7-damage-calc/zhTW/effectiveness.html]: Fixed https://github.com/maphe/maphe.github.io/pull/39/commits/4f92bb0f86b6a0286ceb9256273be8531e1b4cea

**Pull request merged**